src/main/java/org/springframework/data/neo4j/core/mapping/Constants.java

```diff
@@ -31,7 +34,10 @@
 @API(status = Status.EXPERIMENTAL, since = "6.0")
 public final class Constants {
 
-	public static final SymbolicName NAME_OF_ROOT_NODE = Cypher.name("n");
```

Comment:
I would do a `NAME_OF_TYPED_ROOT_NODE.apply(null)` and assigned that, so that it is somewhat compatible in case anyone used those constants.